vavr/src/main/java/io/vavr/control/Either.java

```diff
@@ -163,6 +165,29 @@
     }
 
     /**
+     * Reduces many {@code Either}s into a single {@code Either} by transforming an
+     * {@code Iterable<Either<? extends L, ? extends R>>} into a {@code Either<? extends L, Seq<R>>}. If any of
+     * the {@code Either}s are {@link Either.Left}, then this returns a {@link Either.Left}.
+     *
+     * @param values An {@link Iterable} of {@code Either}s
+     * @param <L>    left type of the Eithers
+     * @param <R>    right type of the Eithers
+     * @return A {@code Either} of a {@link Seq} of results
+     * @throws NullPointerException if {@code values} is null
+     */
+    static <L,R> Either<L, Seq<R>> sequence(Iterable<? extends Either<? extends L, ? extends R>> values) {
+        Objects.requireNonNull(values, "values is null");
+        Vector<R> vector = Vector.empty();
```

Comment:
hmm but in the end we do return the list in case of success. So maybe it's better to create directly the right format rather than paying the conversion cost later? Also note that I took `Try.sequence` as a model, it's done the same there; so if we chnage, we can change both.